### PR TITLE
Fix oldest supported libtpu install command

### PR DESCRIPTION
**Pull request by skye**

### CODE REVIEW

**Comment by skye** [comment]
Still broken after this but getting closer: https://github.com/google/jax/actions/runs/6554208430/job/17800793993

**Comment by skye** [comment]
@sagyakwa

**Comment by skye** [comment]
@jyingl3

**Comment by jyingl3** [comment]
I am working on a fix for the `RuntimeError: Unable to initialize backend 'tpu': module 'libtpu' has no attribute 'get_library_path' (set JAX_PLATFORMS='' to automatically choose an available backend)` https://github.com/google/jax/pull/18192

File: .github/workflows/cloud-tpu-ci-nightly.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
         tpu-type: ["v3-8", "v4-8", "v5e-4"]
     name: "TPU test (jaxlib=${{ matrix.jaxlib-version }}, ${{ matrix.tpu-type }})"
     env:
-      LIBTPU_VERSION_DATE: 20230908
+      LIBTPU_OLDEST_VERSION_DATE: 20230908
```
Comment (jyingl3) [inline .github/workflows/cloud-tpu-ci-nightly.yml:31]: I tested and found that 20230912 is the oldest version that has PJRT API version >= 0.29.

**Review by sagyakwa — APPROVED** [review]

File: .github/workflows/cloud-tpu-ci-nightly.yml (hunk 1 above)
Comment (skye) [reply .github/workflows/cloud-tpu-ci-nightly.yml:31]: Thanks, I'll update the date in this PR too (will be 6 weeks on Tuesday, close enough)

**Pull request merged**